foreman/foreman.spec

```diff
@@ -9,13 +9,13 @@
 %global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
 
 # set and uncomment all three to set alpha tag
-%global alphatag RC1
+%global alphatag RC2
 %global dotalphatag .%{alphatag}
 %global dashalphatag -%{alphatag}
 
 Name:   foreman
 Version: 1.17.0
-Release: 0.3%{?dotalphatag}%{?dist}
+Release: 0.1%{?dotalphatag}%{?dist}
```

Comment:
You should not decrease this because the package will be seen as older since 1.17.0-0.1RC2 < 1.17.0-0.3RC1.